client/src/components/expectations-list.tsx

```diff
@@ -278,11 +281,11 @@ function ExpectationsList(props: {
     updateExpectations([
       ...expectations,
       {
-        expectation: "Add a short ideal answer for an expectation, e.g. 'Red'",
+        expectation: "",
```

Comment:
why remove the default text for the expectation

Reply:
The default text for the expectation caused the user to have to manually highlight and delete/change the text whenever they created a new lesson. This could be a minor obstacle for teachers as usually when you let's say draft an email, there isn't example text already filled out and so by removing the default values we get rid of that minor hiccup for the user and make it more straightforward.